master.cfg

```diff
@@ -370,6 +375,27 @@ for builder in master_config["builders"]:
     if build_type == "rpm":
         properties["rpm_type"] = "".join(os_name.split("-"))
         factory_instance = f_rpm_autobake
+    tags = [os_name, build_type, "autobake"]
+    # From mariadb.org-tools/release/prep - under
+    # Dirs for buildbot.mariadb.org
+    if builder in [
+        "ppc64le-debian-11",
+        "s390x-ubuntu-2004",
+        "s390x-rhel-8",
+        "s390x-sles-12",
+        "s390x-sles-15",
+        "ppc64le-rhel-9",
+        "s390x-rhel-9",
+        "ppc64le-ubuntu-2204",
+        "s390x-ubuntu-2204",
+        "amd64-debian-sid",
+        "aarch64-debian-sid",
+        "ppc64le-debian-sid",
+        "amd64-opensuse-15.5",
+        "amd64-sles-15.5",
+        "s390x-sles-15.5",
+    ]:
+        tags += ["release_packages"]
```

Comment:
This is intended to tag a subset of autobake builders with "release_packages" ?
If yes, this won't work, see line 403: https://github.com/MariaDB/buildbot/blob/643dc81a0cf7cc0299db666ceaa33e9c1b0171c8/master.cfg#L403

Reply:
Or do you want that non-autobake builders in the list to have this tag?
For example this:
![image](https://github.com/user-attachments/assets/d3804101-a538-47b7-8340-e9525ee96d9a)